etc/bash_completion.sh

```diff
@@ -134,6 +134,10 @@ _git_scp(){
   __git_complete_remote_or_refspec
 }
 
+_git_stamp(){
+  __gitcomp '--replace -r'
+}
+
```

Comment:
Alright I'll try to add it since that should be a simple one
I am not sure I'll be able to test it though
Configuring and switching to ZSH is still on my TODO list :)